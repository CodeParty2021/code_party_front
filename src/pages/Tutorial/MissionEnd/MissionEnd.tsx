import React from "react";
import { Link } from "react-router-dom";
type Props = {};

export const MissionEnd: React.FC<Props> = () => {
  return (
    <div>
      <h1>ミッション終了画面</h1>
      <Link to="/tutorial/world/:id/missions">ミッション一覧に戻る</Link>
    </div>
  );
};
