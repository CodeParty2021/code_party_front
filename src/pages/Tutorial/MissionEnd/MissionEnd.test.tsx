import React from "react";
import { shallow } from "enzyme";
import { MissionEnd } from "./MissionEnd";
describe("<MissionEnd />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(<MissionEnd />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
