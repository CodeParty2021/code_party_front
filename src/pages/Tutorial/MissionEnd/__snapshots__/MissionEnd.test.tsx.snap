// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MissionEnd /> auth snapshot test 1`] = `
Array [
  <div>
    <h1>
      ミッション終了画面
    </h1>
    <Link
      to="/tutorial/world/:id/missions"
    >
      ミッション一覧に戻る
    </Link>
  </div>,
]
`;
